mis_builder/models/aep.py

```diff
@@ -283,14 +284,20 @@ def do_queries(self, date_from, date_to,
                                         ['debit', 'credit', 'account_id'],
                                         ['account_id'])
             for acc in accs:
+                company = account_model.browse(acc['account_id'][0]).company_id
                 debit = acc['debit'] or 0.0
                 credit = acc['credit'] or 0.0
                 if mode in (self.MODE_INITIAL, self.MODE_UNALLOCATED) and \
                         float_is_zero(debit-credit,
-                                      precision_rounding=self.dp):
+                            precision_rounding=\
+                            company.currency_id.decimal_places):
                     # in initial mode, ignore accounts with 0 balance
                     continue
-                self._data[key][acc['account_id'][0]] = (debit, credit)
+                if company.currency_id != self.currency_id:
+                    rate = self.currency_id.rate / company.currency_id.rate
```

Comment:
The basic implementation is quite simple.
Get the account for the line, get its company. If account company currency is not the same as APE currency, do the conversion (using current rate for now)